.github/workflows/scripts/healthcheck.sh

```
@@ -0,0 +1,93 @@
+}
+
+# Default container name, can be overridden by environment variable
+CONTAINER_NAME="${CONTAINER_NAME:-talawa-admin-app-container}"
+
+# Cleanup function
+cleanup() {
+    echo "Cleaning up..."
+    local exit_code="${1:-1}"
+    local cleanup_failed=0
+    # Kill any background processes started by this script
+    if !jobs -p | xargs -r kill 2>/dev/null; then
```

Shellcheck (0.10.0)

[error] You are missing a required space after the !.

(SC1035)